tests/test_extension.py

```diff
@@ -1,7 +1,7 @@
 try:
     import unittest.mock as mock
 except ImportError:
-    import mock
+    from unittest import mock
```

Comment:
Do we need this try-except? I imagine a single import should work for all supported Python 3 versions.